package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "5.12.2",
+  "version": "5.12.3",
```

I think this new functionality (more fields in request params) should be a minor version bump rather than a patch:
```suggestion
  "version": "5.13.0",
```

---

Sgtm - what would a patch be then? 🤔

---

A bugfix to existing behavior. For some examples: [v5.11.1](https://github.com/stytchauth/stytch-node/releases/v5.11.1) (bad types), [v3.6.1](https://github.com/stytchauth/stytch-node/releases/v3.6.1) (bad types), [v3.0.2](https://github.com/stytchauth/stytch-node/releases/v3.0.2) (packaging error)